### PR TITLE
Fix +optional in comment for persistentVolumeClaimRetentionPolicy

### PR DESCRIPTION
**Pull request by xrstf**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
I recently randomly noticed a single "+optional" appearing in the generated API docs, which confused me. This PR makes sure the `+optional` is in its own line, like it should be.

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/api/apps/v1/types.go [hunk 1]
```diff
@@ -255,7 +255,8 @@ type StatefulSetSpec struct {
 	// policy allows the lifecycle to be altered, for example by deleting persistent
 	// volume claims when their stateful set is deleted, or when their pod is scaled
 	// down. This requires the StatefulSetAutoDeletePVC feature gate to be enabled,
-	// which is alpha.  +optional
+	// which is alpha.
```
Comment (carlory) [inline staging/src/k8s.io/api/apps/v1/types.go:258]: It's a beta field. https://github.com/kubernetes/enhancements/issues/1847

Comment (xrstf) [reply staging/src/k8s.io/api/apps/v1/types.go]: Fixed :)

**Pull request merged**